CHANGELOG.md

```diff
@@ -9,6 +9,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.8.0] - 2021-03-21
+
+### Added
+
+- Locale independent URL parsing.
```

Comment:
The lines should probably not end with a period, for constancy with the other entries in the change log.

Reply:
Sure. Thank you

```suggestion
- Locale independent URL parsing
```